app/src/main/java/com/swent/assos/ui/screens/profile/Following.kt

```diff
@@ -27,13 +29,19 @@ import com.swent.assos.ui.components.ListItemAsso
 import com.swent.assos.ui.components.LoadingCircle
 import com.swent.assos.ui.components.PageTitleWithGoBack
 
+@SuppressLint("SuspiciousIndentation")
 @OptIn(ExperimentalComposeUiApi::class)
 @Composable
 fun Following(navigationActions: NavigationActions) {
 
   val viewModel: ProfileViewModel = hiltViewModel()
   val followedAssociationsList by viewModel.followedAssociations.collectAsState()
   val loading by viewModel.loading.collectAsState()
+  val update = viewModel.update.collectAsState()
+
+  LaunchedEffect(key1 = Unit) { viewModel.updateUser() }
```

Comment:
Are you sure this Launcher Effect is necessary ? I think keeping only the second suffices.